burningwood/a_sarnak_imitator.pl

```diff
@@ -115,7 +116,7 @@ sub EVENT_ITEM {
 				quest::summonitem(52952);
 				#:: Award AA points
 				$client->AddAAPoints(5);
-    		$client->Message(15,'You have gained 5 ability points!');
+				$client->Message(15,'You have gained 5 ability points!');
```

Comment:
This probably needs expansion flag given AA points wasnt live until Luclin

Reply:
Do we have any reason to believe that prior to Luclin experience was awarded instead? Might be worth digging into...

Reply:
Fixed with [d1583de](https://github.com/Gates-Of-Time/FVProject-Quests/pull/1284/commits/d1583de6ca6aef1261fb8bff6c90c086c1c02c4d); added an expansion check for Luclin.